Create Largest Permutation.cpp

lust-for-code opened the pull request:
Solution of Largest Permutation in Greedy Algorithms section of Hackerrank.

lust-for-code committed:
Create Largest Permutation.cpp

Solution of Largest Permutation in Greedy Algorithms section of Hackerrank.

BhanukaUOM reviewed (changes requested):
please add only  java or python3 solutions.

lust-for-code committed:
Update and rename Largest Permutation.cpp to Largest Permutation.java

Solution of Largest Permutation from Greedy Algorithm section of Hackerrank in java

lust-for-code commented:
I have replaced the code in c++ with code in java.
